src/background/background.ts

```
@@ -1,107 +1,264 @@
+    });
+  }
+
+  private async handleMessage(
+    request: any,
+    sender: chrome.runtime.MessageSender,
+    sendResponse: (response?: any) => void
+  ): Promise<void> {
+    try {
+      switch (request.action) {
+        case 'getActiveTabInfo':
+          const tabs = await chrome.tabs.query({ active: true, currentWindow: true });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)